plutus-ledger-constraints/src/Ledger/Constraints/OffChain.hs

```diff
@@ -642,7 +642,8 @@ processConstraint = \case
 
     MustMintValue mpsHash red tn i -> do
         mintingPolicyScript <- lookupMintingPolicy mpsHash
-        let value = Value.singleton (Value.mpsSymbol mpsHash) tn
+        -- We add zero ada here to be consistent with cardano-ledger behaviour.
```

Comment:
Point to the relevant ledger code? https://github.com/input-output-hk/cardano-ledger/blob/0726ec1fb2bbb3cf434aa46d3501109e2143de35/eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

Unless we shouldn't do that to prevent having broken links in our docs?
Up to you.